advisor/src/main/kotlin/advisors/VulnerableCode.kt

```diff
@@ -49,6 +54,8 @@ private const val BULK_REQUEST_SIZE = 100
  * [VulnerableCode][https://github.com/nexB/vulnerablecode] instance.
  */
 class VulnerableCode(name: String, config: VulnerableCodeConfiguration) : AdviceProvider(name) {
+    companion object : Logging
```

Comment:
Please make this `private` to not leak `Logging` into the API.

Reply:
BTW, it took me a while to figure out that this declaration is needed. If it is missing, you get a strange internal Kotlin compiler error.